index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header id="header" >
+        <span id="logo" >Robertson Arthur</span>
```

Comment:
- [ ] _(optional)_ Great job. As part of the requirement, I recommend putting the logo in a "< a >" tag. In modern websites, the logo usually helps to redirect the home page

Reply:
@m05e5  Thank you. I have implemented it as suggested.